packages/opentelemetry-scope-base/test/noop/scopemanager.test.ts

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
Casing of the test file is inconsistent with the source file, I think this should be like `test/NoopScopeManager.test.ts`.